general/solomiia_tymoshchuk/Task5.java

```diff
@@ -0,0 +1,24 @@
+package general.solomiia_tymoshchuk;
+
+import java.util.Scanner;
+
+/**
+ * Write a program that accepts two numbers and outputs the larger number.
+ * If numbers are equal - output “Numbers are equal”.
+ **/
+public class Task5 {
+
+    void largerNumber() {
+        System.out.println("Enter two numbers");
+        Scanner sc = new Scanner(System.in);
+        float number = sc.nextFloat();
+        float number2 = sc.nextFloat();
+        Object largerNumber = number == number2 ? "Numbers are equal" : Math.max(number, number2);
```

Comment:
nice idea; however, since values have different types (you had to use an object to hold it), I think it might be better to replace the ternary operator with the if-else block.